tests/loader_regression_tests.cpp

```diff
@@ -451,4 +451,45 @@ TEST_F(EnumerateDeviceExtensionProperties, PropertyCountLessThanAvailable) {
     ASSERT_EQ(extension_count, device_extensions.size() - 1);
     ASSERT_TRUE(device_extensions[0].extensionName == enumerated_device_exts[0].extensionName);
     ASSERT_TRUE(device_extensions[0].specVersion == enumerated_device_exts[0].specVersion);
-}
\ No newline at end of file
+}
+#if _WIN32 || _WIN64
+#if _WIN64
+TEST(TryLoadWrongBinary, Win64) {
+    // TODO
```

Comment:
Nit: On these I would suggest:
  1. Create an issue in Github to come back and add test support for Windows.
  2. Add that issue number to each of these todos.
That way we don't lose track.

Reply:
Frankly, I think I'd rather just hammer it out right now. This PR is meant to be able to address the 32/64 bit issue/pr that came in so I think doing it now would be preferable.

Reply:
Ok.  I just didn't want you to have to push this up quickly, walk away, and forget about it.